src/iuse.cpp

```diff
@@ -7478,6 +7599,10 @@ int iuse::camera( player *p, item *it, bool, const tripoint & )
 
 int iuse::ehandcuffs( player *p, item *it, bool t, const tripoint &pos )
 {
+    if( p->is_mounted() ){
```

Comment:
This is called on **every** turn while the item is active, effectively bypassing the logic below. Consider moving it below the `if(t){ ... }` block.

Reply:
Ill just remove it from that function as it appears , this is something tht only happens *to* the player? and not handcuffs that the player themselves can activate.